src/ome_zarr_models/v04/axes.py

```diff
@@ -1,6 +1,17 @@
+from enum import Enum
 from ome_zarr_models.base import Base
 
 
+class AxisType(str, Enum):
```

Comment:
do we need an enum or should we use `typing.getargs` and a literal type? I will try the second option